modules/templates/helper.go

```diff
@@ -176,9 +176,18 @@ func NewFuncMap() template.FuncMap {
 
 		"FilenameIsImage": FilenameIsImage,
 		"TabSizeClass":    TabSizeClass,
+		"BoolPtrVal":      PtrVal[bool],
 	}
 }
 
+// PtrVal return the value of an pointer or init a new empty one based on pointer type
+func PtrVal[T any](ptr *T) T {
```

Comment:
If somebody knows a better method of checking `*bool` type in template ... please tell :)

Reply:
-> `optional.Option[bool]`